src/app/pages/apps/components/app-detail-view/app-resources-card/app-resources-card.component.ts

```diff
@@ -25,7 +25,6 @@ export class AppResourcesCardComponent implements OnInit {
   pool: string;
 
   availableSpace$ = this.dockerStore.selectedPool$.pipe(
-    filter((pool) => !!pool),
     switchMap((pool) => {
       return this.ws.call('pool.dataset.get_instance', [`${pool}/ix-applications`]);
```

Comment:
> So what would it query on this line then?
> 
> It should also be `ix-apps`, there is a constant for it somewhere.

it should query pool from `this.dockerStore.selectedPool$` or some empty value.

Reply:
@undsoft  you mean it should be
`return this.ws.call('pool.dataset.get_instance', [`${pool}/ix-apps`]);`?

Reply:
@undsoft when we want to filter `filter((pool) => !!pool),` we'll not get any response hence infinite skeleton.

Reply:
> it should query pool from this.dockerStore.selectedPool$ or some empty value.

Since `pool` is used in the call, querying `undefined/ix-apps` doesn't seem like a good idea. You should just hide this row if pool is not set.

> return this.ws.call('pool.dataset.get_instance', [${pool}/ix-apps]);?

Yes, search for global constant for `ix-apps`.

Reply:
Okay, got it! Sound good 👍